### PR TITLE
Validate progress

### PR DESCRIPTION
**Pull request by rklaehn**
Also give validate some progress niceness...

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -247,6 +249,111 @@ impl ProgressBarState {
...
+                pb.set_style(ProgressStyle::default_bar().template("{msg}").unwrap());
+                pb.finish_with_message(msg);
+            }
+            //     let mp = self.mp.clone();
+            //     let pb = pb.clone();
+            //     tokio::spawn(async move {
+            //         tokio::time::sleep(Duration::from_millis(500)).await;
+            //         mp.remove(&pb);
+            //     });
+            // }
```
Comment (flub) [inline src/main.rs:350]: ?

File: src/main.rs [hunk 2]
```diff
@@ -247,6 +249,111 @@ impl ProgressBarState {
...
+        }
+    }
+
+    fn all_done(self) {}
```
Comment (flub) [inline src/main.rs:354]: ?

**Review by flub — DISMISSED** [review]

File: src/main.rs (hunk 1 above)
Comment (rklaehn) [reply src/main.rs:350]: Ah well, I tried keeping the successful ones for a second, but it flickered...

File: src/main.rs (hunk 2 above)
Comment (rklaehn) [reply src/main.rs:354]: Turned out that I did not want to do anything on all_done...

**Review by flub — APPROVED** [review]

**Pull request merged**